gitlint/__init__.py

```diff
@@ -202,6 +199,16 @@ def main(argv, stdout=sys.stdout, stderr=sys.stderr):
 
     vcs, repository_root = get_vcs_root()
 
+    error = 'ERROR'
+    skipped = 'SKIPPED'
+    okay = 'OK'
+
+    color = not arguments['--no-color']
```

Comment:
Could you change this to:

```python
color = not arguments['--no-color'] and sys.stdout.isatty()
```

Reply:
I'll close this PR and reopen it with a dedicated branch so other PRs are no longer in it.
I'll also adjust the TODO, but I don't really see how to add a useful test for it.
The git lint command is just called in a shell script by Jenkins. Jenkins only evaluates the return code, but if something goes wrong the developer consults the console output of the Jenkins run and that was rather hard to read with the color codes in it.